clippy_utils/src/msrvs.rs

```diff
@@ -24,7 +24,7 @@ msrv_aliases! {
     1,80,0 { BOX_INTO_ITER }
     1,77,0 { C_STR_LITERALS }
     1,76,0 { PTR_FROM_REF, OPTION_RESULT_INSPECT }
-    1,74,0 { REPR_RUST }
+    1,74,0 { REPR_RUST, IO_ERROR_OTHER }
```

Comment:
Can you also update the `#[lint]` attribute for the `msrv` configuration?

Reply:
Not sure what you mean, the `io_other_error` lint is listed under `#[lints]` in `clippy_config/src/conf.rs` line 610.

Reply:
Nevermind, I'm just blind xD